6001-7000/LOC6554GondarineK.xml

```diff
@@ -120,30 +120,30 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     </note>
                 </place>
                 <listRelation>
-                    <relation name="syriaca:has-relation-to-place"
-                         active="LOC6554GondarineK"
-                         passive="LOC3577Gondar">
+                    <relation name="gn:locatedIn"
+                        active="LOC3577Gondar"
```

Comment:
active-passive?

Reply:
but then the relation does not work, I do not believe it necessarily must be moved to another record as they are harvested everywhere

Reply:
good, let's just wait for confirmation from @PietroLiuzzo , I really want to avoid any more confusion

Reply:
it is ok, they are picked up from all records in the relations view, yes.

Reply:
perfect